snyk/client.py

```diff
@@ -30,35 +34,86 @@ def __init__(
         }
         self.api_post_headers = self.api_headers
         self.api_post_headers["Content-Type"] = "application/json"
+        self.tries = tries
+        self.backoff = backoff
+        self.delay = delay
+
+    def make_http_call(
+        self, method, url: str, headers: object, json={},
+    ) -> requests.Response:
+        resp = method(url, json=json, headers=headers,)
+        if not resp or resp.status_code >= 500:
```

Comment:
for readibility sake, could you replace `500` with `requests.codes.server_error`?

Reply:
done!